utility/src/main/java/org/oppia/util/logging/EventBundleCreator.kt

```diff
@@ -0,0 +1,84 @@
+package org.oppia.util.logging
+
+import android.os.Bundle
+import org.oppia.app.model.EventLog
+import org.oppia.app.model.EventLog.Context.ActivityContextCase.ACTIVITYCONTEXT_NOT_SET
+import org.oppia.app.model.EventLog.Context.ActivityContextCase.EXPLORATION_CONTEXT
+import org.oppia.app.model.EventLog.Context.ActivityContextCase.QUESTION_CONTEXT
+import org.oppia.app.model.EventLog.Context.ActivityContextCase.STORY_CONTEXT
+import org.oppia.app.model.EventLog.Context.ActivityContextCase.TOPIC_CONTEXT
+
+const val TIMESTAMP_KEY = "timestamp"
+const val TOPIC_ID_KEY = "topicId"
+const val STORY_ID_KEY = "storyId"
+const val QUESTION_ID_KEY = "questionId"
+const val EXPLORATION_ID_KEY = "explorationId"
+const val PRIORITY_KEY = "priority"
+
+/**
+ * Utility for creating bundles from [EventLog] objects.
+ * Note that this utility may later upload them to remote services.
+ */
+class EventBundleCreator {
+  private var bundle = Bundle()
+
+  fun createEventBundle(eventLog: EventLog): Bundle {
+    bundle =
+      when (eventLog.context.activityContextCase) {
+        EXPLORATION_CONTEXT -> createExplorationContextBundle(eventLog)
+        QUESTION_CONTEXT -> createQuestionContextBundle(eventLog)
+        STORY_CONTEXT -> createStoryContextBundle(eventLog)
+        TOPIC_CONTEXT -> createTopicContextBundle(eventLog)
+        ACTIVITYCONTEXT_NOT_SET -> createNoContextBundle(eventLog)
+      }
+    return bundle
+  }
+
+  /** Creates a bundle from event having exploration context. */
```

Comment:
Creates a bundle from ... -> Returns a bundle from ...
Here and below

Reply:
Done.